static/js/microphone.js

```diff
@@ -19,6 +19,7 @@ class Microphone {
   // called when user gives permission to use the microphone
   handlePermissionSuccess(stream) {
     const context = new AudioContext({sampleRate: 44100});
+    console.log(context.sampleRate);
```

Comment:
Can you take out print statements? Helpful for when you're debugging but pushing print statements often accumulates when working with others and then eventually no one knows what's being printed out anymore.